src/langcheck/metrics/_pairwise_text_quality_utils.py

```diff
@@ -91,8 +92,18 @@ def enforce_pairwise_comparison_consistency(
         swapped_scores: The scores for the swapped order of the models
         swapped_explanations: The explanations for the swapped order of the
             models
+        score_map: The mapping from the assessment results to the scores (e.g.
+            {'Response A': 0.0, 'Response B': 1.0, 'Tie': 0.5})
     """
-    # Iterate through the scores and explanations to check for consistency.
+    # Get the sorted list of available scores
+    sorted_available_scores = list(score_map.values())
+    sorted_available_scores.sort()
+
+    # Iterate through the scores and explanations to check for consistency. A
+    # score is consistent if the score's index in the sorted_available_scores
```

Comment:
nit: it would be clearer if you could move the sentence from `A score is consistent if...` to line 118. It seems a bit far from the actual logic.

Reply:
makes sense, done!